examples/todo/js/mutations/ChangeTodoStatusMutation.js

```diff
@@ -48,14 +45,11 @@ export default class ChangeTodoStatusMutation extends Relay.Mutation {
     };
   }
   getOptimisticResponse() {
-    var viewerPayload;
-    if (this.props.viewer.todos) {
-      viewerPayload = {id: this.props.viewer.id, todos: {}};
-      if (this.props.viewer.todos.completedCount != null) {
-        viewerPayload.todos.completedCount = this.props.complete
-          ? this.props.viewer.todos.completedCount + 1
-          : this.props.viewer.todos.completedCount - 1;
-      }
+    var viewerPayload = {id: this.props.viewer.id};
+    if (this.props.viewer.completedCount != null) {
+      viewerPayload.completedCount = this.props.complete
+        ? this.props.viewer.completedCount + 1
+        : this.props.viewer.completedCount - 1;
```

Comment:
Huh? I literally didn't change anything but whitespace with the ternary. Do you want all of the cases where this shows up to look like e.g.

```js
      viewerPayload.completedCount = this.props.complete ?
        this.props.viewer.completedCount + 1 :
        this.props.viewer.completedCount - 1;
```